### PR TITLE
Workflow Sample Working with Dataset

### PR DESCRIPTION
**Pull request by zhaocaih**
Signed-off-by: Caihua Zhao <zhaocaih@cn.ibm.com>

### CODE REVIEW

File: zOSMF-Samples/workflow_sample_dataset.xml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+/* See the License for the specific language governing permissions and          */
+/* limitations under the License.                                               */
+/********************************************************************************/
+-->
+
+<workflow>
+    <workflowInfo>
+        <workflowID>workflow.sample.dataset</workflowID>
+		<workflowDefaultName>Sample workflow to manipulate data sets</workflowDefaultName>
+        <workflowDescription>Sample that demonstrates working with a dataset and gether feedback for a step.</workflowDescription>
```
Comment (shreeji818) [inline zOSMF-Samples/workflow_sample_dataset.xml]: <workflowDescription>Sample that demonstrates working with a dataset and gather feedback for a step.</workflowDescription>

**Review by shreeji818 — CHANGES_REQUESTED** [review]
Please update copyright as mentioned in previous review.

**Comment by zhaocaih** [comment]
I updated the copyright, please help review, thanks!

**Comment by zhaocaih** [comment]
Updated the format, please review again, thanks.

**Review by shreeji818 — APPROVED** [review]

**Comment by zhaocaih** [comment]
> Description for file is not right. It picked up the last comment regarding the copyright update.

I am not sure I understand this, do I need to take any actions here?

**Pull request merged**